python_docs_theme/static/pydoctheme.css

```diff
@@ -33,6 +33,31 @@ a[href] {
     text-decoration: underline 1px;
     text-underline-offset: 0.25em;
 }
+/* No underline for navigation */
+a.headerlink,
+div.genindex-jumpbox a,
+div.modindex-jumpbox a,
+div#search-results a,
+div.sphinxsidebar a,
+div.toctree-wrapper a,
+div[role=navigation] a,
+table.contentstable a,
+table.indextable a {
+    text-decoration: none;
+}
+
+/* Except when hovered */
+div.genindex-jumpbox a:hover,
+div.modindex-jumpbox a:hover,
+div#search-results a:hover,
+div.sphinxsidebar a:hover,
+div.toctree-wrapper a:hover,
+div[role=navigation] a:hover,
+table.contentstable a:hover,
+table.indextable a:hover {
+    text-decoration: underline;
+    text-underline-offset: auto;
```

Comment:
`auto` should already be the default -- is there something else overriding this when it's not explicitly stated, or did you add it as a precaution to prevent other rules from overriding the value in case the offset is added to some parent container in the future?

Reply:
> is there something else overriding this when it's not explicitly stated ...

Yes, it's overridden by this:

```css
a[href] {
    text-decoration: underline 1px;
    text-underline-offset: 0.25em;
}
```

So "normal" body text links have underlines with extra offset; but navigational links only have underlines for hover, and they're the default offset.